analysis/__main__.py

```diff
@@ -106,6 +107,19 @@ def summarize_setting(summary, version, seconds, path, data):
             data = data.split(",")[0].split("(")[0].strip()
             data = data.replace("@TM@", "(TM)").replace("@R@", "(R)").replace("@C@", "(C)")
 
+            if "nvidia" in data.lower() or "geforce" in data.lower() or "quadro" in data.lower():
+                brand = "NVIDIA"
+            elif "intel" in data.lower():
+                brand = "Intel"
```

Comment:
Mildly biased here, but maybe worth keeping distinction between intel onboard and intel discrete gpus?

Reply:
If you want to know the details of the Intel entries, use the non-brand table :) I really don't want this to be maintenance heavy. This is simple ;)